src/rgw/rgw_rados.cc

```diff
@@ -9258,7 +9258,8 @@ int RGWRados::get_system_obj(RGWObjectCtx& obj_ctx, RGWRados::SystemObject::Read
                              RGWObjVersionTracker *objv_tracker, rgw_obj& obj,
                              bufferlist& bl, off_t ofs, off_t end,
                              map<string, bufferlist> *attrs,
-                             rgw_cache_entry_info *cache_info)
+                             rgw_cache_entry_info *cache_info,
+			     boost::optional<obj_version>)
```

Comment:
this is not used and I didn't see this change on master. is there a reason?

Reply:
it's from the original changeset.  can strike if  you prefer, as it is terminal at jewel